packages/cardano-services/test/cli.test.ts

```diff
@@ -2133,7 +2133,34 @@ describe('CLI', () => {
           );
           await serverStarted(apiUrl);
           const headers = { 'Content-Type': 'application/json' };
-          const res = await axios.post(`${apiUrl}/${ServiceNames.StakePool}/health`, { headers });
+          const res = await axios.post(`${apiUrl}/${ServiceNames.Asset}/health`, { headers });
```

Comment:
Why was this changed to `Asset`? The test is for `stakepool provider server`

Reply:
Fixed on 8b91155